build.gradle

```diff
@@ -231,12 +242,13 @@ allprojects {
 
     testImplementation group: 'org.postgresql', name: 'postgresql', version: '42.5.0'
     testImplementation group: 'org.testcontainers', name: 'postgresql', version: '1.17.5'
-    testImplementation group: 'io.rest-assured', name: 'xml-path', version: restAssuredVersion
-    testImplementation group: 'io.rest-assured', name:'json-path', version: restAssuredVersion
-    testImplementation group: 'org.codehaus.groovy', name: 'groovy', version: groovyVersion
-    testImplementation group: 'org.codehaus.groovy', name: 'groovy-xml', version: groovyVersion
-    testImplementation group: 'org.codehaus.groovy', name: 'groovy-json', version: groovyVersion
-    testImplementation group: 'com.github.hmcts', name: 'fortify-client', version: '1.2.0:all'
+
+    testImplementation "io.rest-assured:xml-path:${restAssuredVersion}"
+    testImplementation "io.rest-assured:json-path:${restAssuredVersion}"
+    testImplementation "org.codehaus.groovy:groovy:${groovyVersion}"
+    testImplementation "org.codehaus.groovy:groovy-json:${groovyVersion}"
+    testImplementation "org.codehaus.groovy:groovy-xml:${groovyVersion}"
+    testImplementation 'com.github.hmcts:fortify-client:1.2.0:all'
```

Comment:
Yes, it is not able to download the versions which has !! with the old implementation

Reply:
OK.  Makes me wonder whether all of the dependency declarations should use the new notation rather than the old one!  That can be addressed at another time.